client/lib/query-manager/activity/index.js

```diff
@@ -0,0 +1,17 @@
+/** @format */
+/**
+ * Internal dependencies
+ */
+import QueryManager from 'lib/query-manager';
+
+/**
+ * ActivityQueryManager manages Activity which can be queried
+ */
+export default class ActivityQueryManager extends QueryManager {
+	constructor( data, options ) {
+		super( data, {
+			itemKey: 'ts_utc',
```

Comment:
For `ThemeQueryManager`, we also have to set `itemKey`, but we do it at consumer (i.e. reducer) level, https://github.com/Automattic/wp-calypso/blob/8bc7c784d354614b352beb2d05940bcde291dfa7/client/state/themes/reducer.js#L276

(Fortunately, there's only one consumer.)

I'm not sure which is preferable. Conceptually, hard-coding that inside the class definition seems good, but OTOH, not having to do that is alluring, too.